pkg/plugins/manager/testdata/app-with-child/dist/child/plugin.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
```

Comment:
Unfortunately these plugins are signed, so if we update the source we'd need to re-sign them to get a new `MANIFEST.txt`. It's not a big deal to do that, but considering this is test data then it's probably fine to omit.

Reply:
Thanks for taking a look!

Given that I wouldn't plan to do the resigning process myself, would you prefer that I revert all the changes to the `pkg/plugins/manager/testdata/` directory?

Reply:
Yeah I think that's the best option for now 👍

Reply:
Nice! Done in aa323b2e51b